### ISSUE TITLE
Error：ee_install() and ee_Initialize()

### ISSUE THREAD

**Issue by Yinshao-lzu** [r-spatial/rgee#99]
* rgee version:1.0.5
* R version:4.0.2
* Operating System:Win10

Hello, I followed the example to load the RGEE package, but an error occurred.

![ee_install](https://user-images.githubusercontent.com/56902640/91783798-d472c500-ec33-11ea-9ceb-dfe624e5d23a.jpg)
![ee_initialize](https://user-images.githubusercontent.com/56902640/91783807-db013c80-ec33-11ea-9a90-e02c59c36ad6.jpg)

Please teach me how to solve this problem.

**Comment by francofrolla** [comment]
Do you try: 

library(rgee)
library(reticulate)
earthengine_python <- Sys.getenv("EARTHENGINE_PYTHON", unset = NA)
Sys.setenv(RETICULATE_PYTHON = earthengine_python)
reticulate::py_config()

#RESTART R and try ee_Initialize() 

https://github.com/r-spatial/rgee/issues/91

**Comment by csaybar** [comment]
Hi @Yinshao-lzu sorry for the late reply.

`rgee` depends on `reticulate` and `reticulate` needs that windows users use [miniconda](https://rstudio.github.io/reticulate/articles/python_packages.html ).  I going to add more information in `ee_install` in the next release of rgee. So to solve please first install miniconda after that makes something like this:

```
library(reticulate)

# 1. Terminate your are R session
# 2. Install miniconda
# install_miniconda() 
# 3. Set your miniconda Python Env
use_python("PUT_HERE_THE_PYTHON_PATH_THAT_MINICONDA_CREATE")
# 4. Run ee_install()
ee_install()
```

**Comment by guicastrol** [comment]
Hi @csaybar. Congratulations for the awesome work with rgee.
I still having some issues though.
After following the discussions [here](https://github.com/r-spatial/rgee/issues/91) and [here](https://github.com/r-spatial/rgee/issues/17), I installed and used rgee perfectly. However, after restarting my computer I still getting the same status on  ee_check() :   'earthengine-api not installed'. 

whenever I run ee_Initialize(), I find this error message:

> Error in ee_connect_to_py(ee_current_version, n = 5) : 
>  The current Python PATH: C:/Users/Guilherme/AppData/Local/r-miniconda/envs/rgee/python.exe
> does not have the earthengine-api installed. Are you restarted/terminated your R session?.
> If no, please try:
> ee_install(): To create and set a Python environment with all rgee dependencies.
> ee_install_set_pyenv(): To set a specific Python environment.

This is not supposed to happen, once I have everything installed already  . What am I doing wrong?

Some info:
___________
> Sys.getenv('EARTHENGINE_PYTHON')
> [1] "C:\\Users\\Guilherme\\AppData\\Local\\r-miniconda\\envs\\rgee\\python.exe"
> Sys.getenv("EARTHENGINE_ENV")
>[1] "rgee"

> ee_check() 
> (*)  Python version
> √ [Ok] C:/Users/Guilherme/AppData/Local/r-miniconda/envs/rgee/python.exe v3.8
> √ [X] earthengine-api not installed

> conda_list()
>          name
> 1 r-reticulate
> 2         rgee
>                                                                             python
> 1 C:\\Users\\Guilherme\\AppData\\Local\\r-miniconda\\envs\\r-reticulate\\python.exe
> 2         C:\\Users\\Guilherme\\AppData\\Local\\r-miniconda\\envs\\rgee\\python.exe


> earthengine_python <- Sys.getenv("EARTHENGINE_PYTHON", unset = NA)
> Sys.setenv('RETICULATE_PYTHON' = earthengine_python)
> system.file("python/ee_utils.py", package = "rgee")
[1] "C:/Users/Guilherme/Documents/R/win-library/4.0/rgee/python/ee_utils.py"
> ee_current_version <- system.file("python/ee_utils.py", package = "rgee")
> ee_utils <- rgee::ee_source_python(ee_current_version)
Erro: 'ee_source_python' não é um objeto exportado de 'namespace:rgee'
> traceback()
3: stop(gettextf("'%s' is not an exported object from 'namespace:%s'", 
       name, getNamespaceName(ns)), call. = FALSE, domain = NA)
2: getExportedValue(pkg, name)
1: rgee::ee_source_python

It seems that ee_source_python() is not available in rgee anymore.

> ee_current_version
> [1] "C:/Users/Guilherme/Documents/R/win-library/4.0/rgee/python/ee_utils.py"

**Comment by csaybar** [comment]
Hi @guicastrol did you check #110? It seems the Python API is not installed in your current Python Env

**Comment by guicastrol** [comment]
> Hi @guicastrol did you check #110? It seems the Python API is not installed in your current Python Env

Cesar, I'm using RStudio Version 1.3.1073. It's not possible to do it like you suggested in #110 
Is There another way to set reticulate environment?

**Comment by guicastrol** [comment]
@csaybar , thank you.
I've upgradade Rstudio to version 1.4 and changed the Python interpreter in Tools > Project options
In the end, the point is that we must set the same python interpreter that R is using, right? 

I mean, the path I get with:

>Sys.getenv('EARTHENGINE_PYTHON')
> [1] "C:\\Users\\Guilherme\\AppData\\Local\\r-miniconda\\envs\\rgee\\python.exe"

must be the same I set in the R project options:

![image](https://user-images.githubusercontent.com/72578825/98715710-4a4c8880-2369-11eb-975f-73c83a321526.png)

**Comment by csaybar** [comment]
Hi @guicastrol,
Now is only necessary to set a Python interpreter (I mean at Tools > Project options > Python) that has installed the **Earth Engine Python API**!

**Comment by guicastrol** [comment]
@csaybar , Yes, that's it. Thank you

**Comment by mcket747econ** [comment]
Hello. I have been running into the same issues mentioned in this discussion. When I run ee_install(py_env="rgee")
I get:
![image](https://user-images.githubusercontent.com/59522800/149200716-763a4600-7bfc-433a-9f88-67e83bd4a24b.png).

I then follow the instructions and get the same issue:
![image](https://user-images.githubusercontent.com/59522800/149200848-f33d2cc1-8419-4848-8d4f-c6dabd754924.png)

I have already installed miniconda and I have attempted ( and I think succeeded) in installing the earth engine-api but each time I then try to use it I get an error saying that the earth-engine api is not installed at the python path that I am using. I am not sure how to install the earth engine api to the correct path (I am very new to earth engine) and still a relative beginner with R. Some advice on how to fix this would be appreciated. 

This is the error I get when I run my main code: 

![image](https://user-images.githubusercontent.com/59522800/149201498-3750be9b-5833-4256-a517-f7ac6234f392.png)


And these are the various python interpreters I have. 
![image](https://user-images.githubusercontent.com/59522800/149201648-5f57caf5-c88b-4a4a-9c75-40bab93fafdd.png)

Thanks!
